### ISSUE TITLE
NOT AN ISSUE: SUGGESTION

### ISSUE THREAD

**Issue by 1kali2kali** [Oros42/IMSI-catcher#7]
Sir will it be nice to suggest me how to combine the info en IMSI with the find_cell so I can display the MCC/MNC/LCC info with the IMSI located, can you please ? Thanks in advance

**Comment by Oros42** [comment]
Hello
The easy way :
All informations of find_cell_id.py are link to the frequency (and your location). Same things for simple_IMSI-catcher.py.
So if you run simple_IMSI-catcher.py and find_cell_id.py in parallel at one frequency then all IMSI you got have the same cell tower (same MCC/MNC/LCC).

Complicate way :
Put the code of find_cell from find_cell_id.py in find_imsi from simple_IMSI-catcher.py after ligne 188.
And fix print().

**Comment by 1kali2kali** [comment]
sweeeet I will got thru complicate way.... sounds better !! Thanks !!

**Comment by Oros42** [comment]
Okay :-)
Please let me know how you will display MCC/MNC/LCC.
I am interested ;-)

**Comment by oscarmh** [comment]
wow its more tricky than I suppose......

**Comment by 1kali2kali** [comment]
sure..... what´s you approach ?

**Comment by 1kali2kali** [comment]
![noob](https://cloud.githubusercontent.com/assets/27701381/25105312/4e4b1be8-23c4-11e7-8f1e-8c8cbf951f87.PNG)
N00b!!!! More tomorrow !!!

**Comment by 1kali2kali** [comment]
Blocked.... any suggestion for show_imsi in order to include MCC/MNC/LAC ??

**Comment by Oros42** [comment]
Cell informations are not send every time so you can store MCC/MNC/LAC in a global variable.
But if you change the frequency, MCC/MNC/LAC will not be immediately update :-/

**Comment by 1kali2kali** [comment]
![captura](https://cloud.githubusercontent.com/assets/27701381/25128859/ba1356e8-243b-11e7-80fe-8a923cc67d0f.PNG)
;-) working !! and yessss changes LAC/CellID when change freq

**Comment by Oros42** [comment]
Good job :-)
Could you share your code ?

**Comment by 1kali2kali** [comment]
oh my gosh master ask for it........ attached !! 
[imsipluscellfinder.py.txt](https://github.com/Oros42/IMSI-catcher/files/930103/imsipluscellfinder.py.txt)

**Comment by Oros42** [comment]
Thanks.
I will update my code soon :-)

**Comment by Oros42** [comment]
Merge done.
But what I say previously, it's not perfect when you change the frequency.
![bug](https://cloud.githubusercontent.com/assets/3344155/25141359/9f5b4630-2463-11e7-8856-7d205b1ea691.png)

**Comment by 1kali2kali** [comment]
d#mn something we can do on that ?.....

**Comment by Oros42** [comment]
Perhaps catching the switch of frequency from airprobe_rtlsdr.py ?
Or patch airprobe_rtlsdr.py for sending the frequency on network ?

**Comment by E3V3A** [comment]
@Oros42 

> Perhaps catching the switch of frequency from airprobe_rtlsdr.py ?
> Or patch airprobe_rtlsdr.py for sending the frequency on network ?

Why did you close this? Did you actually implement it?

**Comment by Oros42** [comment]
It's not related to the initial question and the issue is old without activity.
So this points need a new issue. But I have no idea of who to resolve it.
